### PR TITLE
Added firebase authentication to the website

### PR DESCRIPTION
**Pull request by sumitkr2000**
Hey, @royalpinto007 I have successfully added firebase authenticate to the website. The features of authentication are:-

1. If the user is not signed in he/she is redirected to the login page.
2. If the user is on the login or signup page he/she can't access any website page.
3. If the user is successfully logged in he/she cannot move to the login/signup page.
4. I have also implemented the logout functionality on the website.

Please review my pr and consider it a level 3. Thank you!

## Screen Recording

[screen-capture (1).webm](https://github.com/Stoccoin-Official/Stoccoin-Website/assets/100675296/05c8b2cb-feeb-4f78-9dae-6a8801e74876)

### CODE REVIEW

File: client/src/firebase/auth.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+
+const firebaseConfig = {
```
Comment (royalpinto007) [inline client/src/firebase/auth.js:4]: Please remove your data from here :)

**Review by royalpinto007 — CHANGES_REQUESTED** [review]
Please go through the file that I have asked to make changes to, thanks for understanding, also don't forget to delete this project from your firebase account, after you delete these changes from the file, as it can be dangerous for you too :)

**Review by royalpinto007 — APPROVED** [review]

**Comment by royalpinto007** [comment]
Thank you for this @sumitkr2000 :)

**Comment by sumitkr2000** [comment]
@royalpinto007 okay bro. Please make a project in your firebase account and create a .env file and add all the variables into it.

**Comment by sumitkr2000** [comment]
Or maybe update the readme file sk that contributors create their own api keys before starting the project

**Pull request merged**